.github/workflows/ci.yml

```diff
@@ -106,6 +106,13 @@ jobs:
           echo "::set-output name=version::$($xml.project.parent.version)"
           echo "::set-output name=class::$($xml.project.build.plugins.plugin.executions.execution.configuration.transformers.transformer.mainClass)"
         shell: pwsh
+      - name: Set tags
```

Comment:
just small lexical change, maybe something bit more precise like "Set latest tag"

Reply:
cool catch, it makes sense to have it here and at 184 line, pushing with next commit

Reply:
seeing similar ones bellow make me wonder if this is applicable ^
i wouldn't complicate matter further, let it be as is ⚡